### DISCUSSION TITLE
Request: Video on Demand

### DISCUSSION THREAD

**Discussion by joaquinito2048** [iptv-org/community#256] [Ideas]
Hello.
We need in this repository all movies and all episodes of tv shows on Video on Demand available in the internet.
We need all these in all languages, such as English, Spanish, and others.
Paid IPTV providers have Video on Demand, but in this repository, it will include the same as the paid IPTV providers, and also it will include, the BitTorrent files over HTTP, the eDonkey2000 files over HTTP, and the Gnutella files over HTTP.
BitTorrent, eDonkey2000, and Gnutella files will be included (and downloaded) to HTTP server, to watch anywhere and anytime without the need to downloading from computers.
Thank you.

**Comment by Dum4G** [comment]
Do you know any sources of VODs that are not violating any copyrights and available to anyone? If so that could be another repository. IPTV-ORG has grown as such because the content here is coming from legal sources and lasts more than one day

**Comment by BellezaEmporium** [comment]
Hmm...

Well this repository is mainly TV channels, having VoD would not match the philosophy of this repository...

Basically we would only accept public domain movies + no-copyright content.

**Comment by joaquinito2048** [comment]
heroku-miraheze, and me, we created an new repository for VOD sources, 100% legally, directly distributed via legal and free VOD platforms (mostly Pluto TV).
https://github.com/VOD-Org/vod
You are welcome to contribute in this repository, and add shows and movies, in M3U8, or MP4, or any format.
You are invited to the new repository, and in organization.
